File: OCH_Tao/ABC/B/050.py
N = int(input())
T = list(map(int,input().split()))
M = int(input())
l = []
for i in range(M):
  p,x = map(int,input().split())
  l.append(sum(T)-T[p-1]+x)
print("\n".join(str(i) for i in l))

File: OCH_Tao/ABC/B/050.js
const X = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const N = parseInt(X.shift(), 10);
const T = (X.shift().split(" ")).map(x => parseInt(x, 10));
const M = parseInt(X.shift(), 10);
const SUM = T.reduce((a, b) => a + b);
for (let i = 0; i < M; i++) {
  let [p, x] = (X[i].split(" ")).map(x => parseInt(x, 10));
  console.log(SUM - T[p - 1] + x);
}